custom/icds_reports/management/commands/imbalanced_indicator.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import, print_function
+
+from __future__ import unicode_literals
+
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+PERSON_TABLE_ID = 'static-person_cases_v2'
+AWC_LOCATION_TABLE_ID = 'static-awc_location'
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        indicator_list = self.get_indicator_list()
+
+        for indicator in indicator_list:
+            indicator_result = self.execute_query(indicator)
+
+            for ind in indicator_result:
+                print("{}, {}, {}, {}".format(ind[0], indicator['num'], ind[1], ind[2]))
+
+    def execute_query(self, params):
+        query = 'select month,({0}) as numerator, {1} as denomerator \
+        from {2} where  aggregation_level=1   group by month having ({0})>{1};'
+        query_with_param = query.format(*(params['num'], params['denom'], params['table']))
+        with connection.cursor() as cursor:
+            cursor.execute(query_with_param)
+            return cursor.fetchall()
+
+    def get_indicator_list(self):
+        indicator_list = [
```

Comment:
It would be great if you could group these by table, so that we can do one query against each of the agg tables, instead of one query per indicator. Otherwise this will be quite slow

Reply:
All looks, quite easy except this one. Probably will bother you today for this ;)

Reply:
done